src/cdk/overlay/keyboard/overlay-keyboard-dispatcher.ts

```diff
@@ -85,8 +73,8 @@ export class OverlayKeyboardDispatcher implements OnDestroy {
   /** Keyboard event listener that will be attached to the body. */
   private _keydownListener = (event: KeyboardEvent) => {
     if (this._attachedOverlays.length) {
-      // Dispatch keydown event to the correct overlay.
-      this._selectOverlayFromEvent(event)._keydownEvents.next(event);
+      // Dispatch the keydown event to the top overlay.
```

Comment:
Expand this with the rationale for _why_ we always use the most recently opened overlay?